cmd/main.go

```diff
@@ -130,6 +132,16 @@ func (u *UserInterface) Type() string {
 	return "UserInterface"
 }
 
+var defaultToolConfigPaths = []string{
+	filepath.Join("{CONFIG}", "kubectl-ai", "tools.yaml"),
+	filepath.Join("{HOME}", ".config", "kubectl-ai", "tools.yaml"),
+}
+
+var defaultConfigPaths = []string{
+	filepath.Join("{CONFIG}", "kubectl-ai", "config.yaml"),
+	filepath.Join("{HOME}", ".config", "kubectl-ai", "config.yaml"),
+}
```

Comment:
I think we can simplify this by having a single variable named `defaultLookupPaths` and then just appending the config or tools file name.